src/main/java/io/github/syst3ms/skriptparser/expressions/ExprRandomNumber.java

```diff
@@ -0,0 +1,75 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import org.jetbrains.annotations.Nullable;
+import java.util.concurrent.ThreadLocalRandom;
+import java.lang.Math;
+/**
+ * Generate a random number (double) or integer.
+ *
+ * @name random number
+ * @pattern [a] random (integer|number) (from|between) %number% (to|and) %number%
+ * @since ALPHA
+ * @author WeeskyBDW
+ */
+public class ExprRandomNumber implements Expression<Number> {
+
+    private Expression<Number> lowerNumber, maxNumber;
+    private boolean isInteger;
+    private boolean isStictly;
+    private final ThreadLocalRandom thread = ThreadLocalRandom.current();
+
+    static {
+        Main.getMainRegistration().addExpression(
+                ExprRandomNumber.class,
+                Number.class,
+                true,
+                "[a] [(1:strictly)] random integer (from|between) %integer% (to|and) %integer%",
+                "[a] [(1:strictly)] random number (from|between) %number% (to|and) %number%"
+        );
+    }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public boolean init(Expression<?>[] expressions, int matchedPattern, ParseContext context) {
+        lowerNumber = (Expression<Number>) expressions[0];
+        maxNumber = (Expression<Number>) expressions[1];
+        isInteger = matchedPattern == 0;
+        isStictly = context.getParseMark() == 1;
+        return true;
+    }
+
+    @Override
+    public Number[] getValues(TriggerContext ctx) {
+        Number low = lowerNumber.getSingle(ctx);
+        Number max = maxNumber.getSingle(ctx);
+        if (low == null || max == null) {
+            return new Number[0];
+        }
+        int lowInt = low.intValue();
+        int maxInt = max.intValue();
+        if (lowInt > maxInt) {
+            int oldMax = maxInt;
+            maxInt = lowInt;
+            lowInt = oldMax;
+        }
+        if (isStictly) {
+            lowInt--;
```

Comment:
```suggestion
            lowInt++;
            maxInt--;
```
This is the desired behavior.